doc/content/examples/example11.md

```diff
@@ -0,0 +1,104 @@
+# Example 11: Cantilever Bending with Continuum Elements - Mesh Refinement and Element Type
+
+This example demonstrates a 3D cantilever beam subject to static and dynamic loading. Currently, the shell elements available for use in MASTODON are not as robust as continuum elements, particularly when it comes to dynamic simulations. This example was created to show some modeling best practices and things to be aware of when simulating structures or other 3D geometry in dynamic, high-bending type scenarios - particularly when continuum elements are used in lieu of shell elements.
```

Comment:
Can you remove the sentence on shells? We can just say this is about modeling bending using continuum elements.